src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -435,9 +435,9 @@ public void CheckOldResourceDesignerIsNotUsed (bool isRelease, ProjectLanguage l
 				if (File.Exists (designer))
 					File.Delete (Path.Combine (Root, b.ProjectDirectory, designer));
 				Assert.IsTrue (b.Build (proj), "Build should have succeeded.");
-				Assert.IsFalse (File.Exists (Path.Combine (b.ProjectDirectory, "Resources",
-					"Resource.designer"  + proj.Language.DefaultDesignerExtension)),
-					"{0} should not exists", designer);
+				var fi = new FileInfo (Path.Combine (Root, b.ProjectDirectory, designer));
+				Assert.IsFalse (fi.Length > 3,
```

Comment:
The `designer` file won't/can't contain `<UTF-8 BOM><newline>`? 5 might be a safer value, but `new[]{0xef, 0xbb, 0xbf, 0x0d, 0x0a}.Length` might be a more *understandable* value.